trio/_core/tests/test_local.py

```diff
@@ -92,7 +92,7 @@ async def task2(tok):
 
             n.start_soon(task2, token)
             await _core.wait_all_tasks_blocked()
-            assert t1.get() == "cod"
+            assert t1.get() == "swordfish"
```

Comment:
it wasn't clear that this expectation corresponds to the `set()` in task2, rather than the `set()` at line 86